source/versions.json

```diff
@@ -1,74 +1,2827 @@
+      "sunos-x64",
+      "sunos-x86",
+      "win-x64-exe",
+      "win-x86-exe",
+      "win-x86-msi"
+    ],
+    "npm": "1.4.3",
+    "v8": "3.14.5.9",
+    "uv": "0.10.25",
+    "zlib": "1.2.3",
+    "openssl": "1.0.1e",
+    "modules": "11",
+    "url": "https://nodejs.org/dist/v0.10.26/",
+    "name": "Node.js"
+  },
+  {
+    "version": "v0.10.25",
+    "date": "2014-04-11",
+    "files": [
+      "linux-x64",
+      "linux-x86",
+      "osx-x64-pkg",
+      "osx-x64-tar",
+      "osx-x86-tar",
+      "src",
+      "sunos-x64",
+      "sunos-x86",
+      "win-x64-exe",
+      "win-x86-exe",
+      "win-x86-msi"
+    ],
+    "npm": "1.3.24",
+    "v8": "3.14.5.9",
+    "uv": "0.10.23",
+    "zlib": "1.2.3",
+    "openssl": "1.0.1e",
+    "modules": "11",
+    "url": "https://nodejs.org/dist/v0.10.25/",
+    "name": "Node.js"
+  },
+  {
+    "version": "v0.10.24",
+    "date": "2013-12-19",
+    "files": [
+      "linux-x64",
+      "linux-x86",
+      "osx-x64-pkg",
+      "osx-x64-tar",
+      "osx-x86-tar",
+      "src",
+      "sunos-x64",
+      "sunos-x86",
+      "win-x64-exe",
+      "win-x86-exe",
+      "win-x86-msi"
+    ],
+    "npm": "1.3.21",
+    "v8": "3.14.5.9",
+    "uv": "0.10.21",
+    "zlib": "1.2.3",
+    "openssl": "1.0.1e",
+    "modules": "11",
+    "url": "https://nodejs.org/dist/v0.10.24/",
+    "name": "Node.js"
+  },
+  {
+    "version": "v0.10.23",
+    "date": "2013-12-12",
+    "files": [
+      "linux-x64",
+      "linux-x86",
+      "osx-x64-pkg",
+      "osx-x64-tar",
+      "osx-x86-tar",
+      "src",
+      "sunos-x64",
+      "sunos-x86",
+      "win-x64-exe",
+      "win-x86-exe",
+      "win-x86-msi"
+    ],
+    "npm": "1.3.17",
+    "v8": "3.14.5.9",
+    "uv": "0.10.20",
+    "zlib": "1.2.3",
+    "openssl": "1.0.1e",
+    "modules": "11",
+    "url": "https://nodejs.org/dist/v0.10.23/",
+    "name": "Node.js"
+  },
+  {
+    "version": "v0.10.22",
+    "date": "2013-12-02",
+    "files": [
+      "linux-x64",
+      "linux-x86",
+      "osx-x64-pkg",
+      "osx-x64-tar",
+      "osx-x86-tar",
+      "src",
+      "sunos-x64",
+      "sunos-x86",
+      "win-x64-exe",
+      "win-x86-exe",
+      "win-x86-msi"
+    ],
+    "npm": "1.3.14",
+    "v8": "3.14.5.9",
+    "uv": "0.10.19",
+    "zlib": "1.2.3",
+    "openssl": "1.0.1e",
+    "modules": "11",
+    "url": "https://nodejs.org/dist/v0.10.22/",
+    "name": "Node.js"
+  },
+  {
+    "version": "v0.10.21",
+    "date": "2013-10-18",
+    "files": [
+      "linux-x64",
+      "linux-x86",
+      "osx-x64-pkg",
+      "osx-x64-tar",
+      "osx-x86-tar",
+      "src",
+      "sunos-x64",
+      "sunos-x86",
+      "win-x64-exe",
+      "win-x86-exe",
+      "win-x86-msi"
+    ],
+    "npm": "1.3.11",
+    "v8": "3.14.5.9",
+    "uv": "0.10.18",
+    "zlib": "1.2.3",
+    "openssl": "1.0.1e",
+    "modules": "11",
+    "url": "https://nodejs.org/dist/v0.10.21/",
+    "name": "Node.js"
+  },
+  {
+    "version": "v0.10.20",
+    "date": 
```

Comment:
We need all release for the release history page: https://nodejs.org/en/download/releases/